actionview/lib/action_view/helpers/asset_tag_helper.rb

```diff
@@ -331,18 +335,21 @@ def audio_tag(*sources)
       private
         def multiple_sources_tag_builder(type, sources)
           options       = sources.extract_options!.symbolize_keys
+          fallback      = options.delete(:fallback)
           skip_pipeline = options.delete(:skip_pipeline)
           sources.flatten!
 
           yield options if block_given?
 
           if sources.size > 1
             content_tag(type, options) do
-              safe_join sources.map { |source| tag("source", src: send("path_to_#{type}", source, skip_pipeline: skip_pipeline)) }
+              safe_join sources.map { |source| tag("source", src: send("path_to_#{type}", source, skip_pipeline: skip_pipeline)) }, fallback
```

Comment:
We need a test to this case. Because how it is implemented right now it is going to add multiple fallbacks and I think that is not what we want right?